src/main/webapp/js/testresult.js

```diff
@@ -8,12 +9,13 @@ function reset(){
     resetCharts();
 }
 
-function populateTemplate(){
+function populateTemplate(displayValues){
     reset();
+    displayValuesFlag = displayValues;
     var noOfBuilds = $j('#noofbuilds').val();
     remoteAction.getTreeResult(noOfBuilds,$j.proxy(function(t) {
         var itemsResponse = t.responseObject();
-        treeMarkup = analyzerTemplate(itemsResponse);
+        treeMarkup = analyzerTemplate(itemsResponse, displayValues);
```

Comment:
This is not required. Since in the template we are referring to the global variable.